### PR TITLE
Upgrade to primer 8

### LINKED ISSUE #994: Flash message is a bit less wide than it used to be
It looks off not being the full width of the body below.

screen shot 2017-07-05 at 1 57 42 pm

### PR DESCRIPTION
**Pull request by wilfriedE**
Primer 8 upgrade with a few fixes for mobile.

Resolves #994 , #995 , #996

### CODE REVIEW

File: app/views/organizations/_organization_banner.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% content_for :organization_banner do %>
   <div class="organization-banner" style="background-image: <%= @organization.github_organization.geo_pattern_data_uri %>">
-    <div class="container">
+    <div class=container-lg>
```
Comment (jasonlong) [inline app/views/organizations/_organization_banner.html.erb]: Missing quotes here.

File: app/views/organizations/_organization_invitation_banner.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <% content_for :organization_invitation_banner do %>
   <div class="organization-banner" style="background-image: <%= organization.github_organization.geo_pattern_data_uri %>">
-    <div class="container">
+    <div class=container-lg>
```
Comment (jasonlong) [inline app/views/organizations/_organization_invitation_banner.html.erb]: Missing quotes.

File: app/views/peek/_bar.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <% if peek_enabled? %>
   <div id="peek" class="<%= Peek.env %><%= ' disabled' if cookies[:peek] == 'false' %>" data-request-id="<%= Peek.request_id %>">
-    <div class="container">
+    <div class=container-lg>
```
Comment (jasonlong) [inline app/views/peek/_bar.html.erb]: And here.

File: app/views/shared/_invitation_header.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <header class="site-header">
-  <div class="container">
+  <div class=container-lg>
```
Comment (jasonlong) [inline app/views/shared/_invitation_header.html.erb]: And here. 😺

File: app/views/shared/_message_of_the_day.html.erb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <% if ENV['MOTD'].present? %>
   <div class="message-of-the-day">
-    <div class="container">
+    <div class=container-lg>
```
Comment (jasonlong) [inline app/views/shared/_message_of_the_day.html.erb]: 😸

**Comment by wilfriedE** [comment]
Thanks @jasonlong, should be ✨  now

**Comment by wilfriedE** [comment]
yarn.lock is like Gemfile.lock for the npm packages. I think most of those changes are due to the newer version of `primer-css`.

**Pull request merged**